Src/DependencyCollector/Shared/Implementation/HttpParsers/DocumentDbHttpParser.cs

```diff
@@ -0,0 +1,196 @@
+            ["PUT dbs/*/colls/*/sprocs/*"] = "Replace Stored Procedure",
+            ["GET dbs/*/colls/*/sprocs"] = "List Stored Procedures",
+            ["DELETE dbs/*/colls/*/sprocs/*"] = "Delete Stored Procedure",
+            ["POST dbs/*/colls/*/sprocs/*"] = "Execute Stored Procedure",
+            // User defined function operations
+            ["POST dbs/*/colls/*/udfs"] = "Create UDF",
+            ["PUT dbs/*/colls/*/udfs/*"] = "Replace UDF",
+            ["GET dbs/*/colls/*/udfs"] = "List UDFs",
+            ["DELETE dbs/*/colls/*/udfs/*"] = "Delete UDF",
+            // Trigger operations
+            ["POST dbs/*/colls/*/triggers"] = "Create Trigger",
+            ["PUT dbs/*/colls/*/triggers/*"] = "Replace Trigger",
+            ["GET dbs/*/colls/*/triggers"] = "List Triggers",
+            ["DELETE dbs/*/colls/*/triggers/*"] = "Delete Trigger",
+            // User operations
+            ["POST dbs/*/users"] = "Create User",
+            ["GET dbs/*/users"] = "List Users",
+            ["GET dbs/*/users/*"] = "Get User",
+            ["DELETE dbs/*/users/*"] = "Delete User",
+            ["PUT dbs/*/users/*"] = "Replace User",
+            // Permission operations
+            ["POST dbs/*/users/*/permissions"] = "Create Permission",
+            ["GET dbs/*/users/*/permissions"] = "List Permission",
+            ["GET dbs/*/users/*/permissions/*"] = "Get Permission",
+            ["DELETE dbs/*/users/*/permissions/*"] = "Delete Permission",
+            ["PUT dbs/*/users/*/permissions/*"] = "Replace Permission",
+            // Offer operations
+            ["POST offers"] = "Query Offers",
+            ["GET offers"] = "List Offers",
+            ["GET offers/*"] = "Get Offer",
+            ["PUT offers/*"] = "Replace Offer"
+        };
+
+        /// <summary>
+        /// Tries parsing given dependency telemetry item. 
+        /// </summary>
+        /// <param name="httpDependency">Dependency item to parse. It is expected to be of HTTP type.</param>
+        /// <returns><code>true</code> if successfully parsed dependency.</returns>
+        internal static bool TryParse(ref DependencyTelemetry httpDependency)
+        {
+            string name = httpDependency.Name;
+            string host = httpDependency.Target;
+            string url = httpDependency.Data;
+
+            if (name == null || host == null || url == null)
+            {
+                return false;
+            }
+
+            if (!host.EndsWith(".documents.azure.com", StringComparison.OrdinalIgnoreCase))
+            {
+                return false;
+            }
+
+            ////
+            //// DocumentDB REST API: https://docs.microsoft.com/en-us/rest/api/documentdb/
+            ////
+
+            string account = host.Substring(0, host.IndexOf('.'));
+
+            string verb = null;
+            string nameWithoutVerb = name;
+
+            for (int i = 0; i < DocumentDbVerbPrefixes.Length; i++)
```

Comment:
you can skip the loop if the name starts with `/`. It will mean that we didn't collect VERB for sure and it is a common case without the profiler